phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -261,6 +262,7 @@
 import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
 import com.google.common.primitives.Ints;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
nit: this import doesn't seem to be used

Reply:
done.